.github/ISSUE_TEMPLATE/coblocks-and-or-gutenberg-update-github-tracking-issue-template.md

```diff
@@ -0,0 +1,39 @@
+---
+name: 'CoBlocks and or Gutenberg update GitHub Tracking Issue template'
+about: 'Template for Gutenberg/Coblocks plugin update process'' GitHub tracking issue,
+  more: '
+title: Update Gutenberg to v<version> and CoBlocks to v<version>
+labels: ''
+assignees: ''
+
+---
+
+We have installed Gutenberg `v<version>` and CoBlocks `v<version>` to edge sites for early feedback. 
+
+Feedback and bugs and the final activation of the plugins on wpcom will be tracked here, for anyone interested.
+
+Testing post: p7DVsv-8Oh-p2
+
+### Process
+
+* [ ] Install version 8.3.0-rc.1. (D44703-code)
```

Comment:
```suggestion
* [ ] Install RC (D44703-code)
```
I wonder if it's even helpful to refer to the RC version here. 🤔 you'd have to add extra todo items if you install multiple RCs anyways. (then people can fill in the version as needed)